src/main/java/com/future/function/qa/util/Path.java

```diff
@@ -13,4 +13,8 @@
   String QUESTION_BANK = "/api/scoring/question-banks";
 
   String QUESTION = "/questions";
+
+  String QUIZ = "/api/scoring/batches/%s/quizzes";
```

Comment:
I see from PR #99 such format is not used. Maybe for better consistency, either this PR could follow #99, or change #99 to follow this PR's format